src/odemis/gui/comp/overlay/view.py

```diff
@@ -1343,6 +1344,8 @@ def __init__(self, cnvs, history_list_va):
         self.history = history_list_va  # ListVA  of (center, size) tuples
         self.history.subscribe(self._on_history_update)
 
+        self.merge_ratio = None
+
```

Comment:
It's better to put a merge_ratio "private" by naming "_merge_ratio", and add a set_merge_ratio() method. Then it's clear that it's possible to change it, and also, it will take care of requesting a overlay redraw. Alternatively, you could put it in a property, and add could comment that this is a modifiable attribute.